modules/ROOT/pages/working-with-sandboxes.adoc

```diff
@@ -0,0 +1,266 @@
+=  Working With Sandboxes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use Salesforce sandbox organizations to test your portal in a sandbox environment under the same conditions as your production environment. API Experience Hub enables you to manage separate deployment environments for your API portal. You can connect to multiple Salesforce sandbox organizations and switch between them. Manage Salesforce organizations from the *Manage your API portal* page by selecting sandbox environments that are available in the current Salesforce connection.  
+
+== Before You Begin
+Ensure that you have sandbox organizations already set up. For more information about creating sandboxes, see https://help.salesforce.com/s/articleView?id=sf.data_sandbox_create.htm&type=5[Create a Sandbox] in the Salesforce documentation.
+
+== Manage Salesforce Organization Connections
+
+. In the *Manage your API portal* page, click the connection control. 
++
+image::aeh-org-switcher.png["Connection control",15%]
+. To connect to a production organization, click *Connect production org* from the *Connected Salesforce Orgs* dialog. 
++
+The connection process restarts and enables you to connect to the production organization.
+. To switch to another a sandbox for the connected production organization, click *Switch to this org* from the *Connected Salesforce Orgs* dialog.
++
+The page displays the metadata for the sandbox organization.
+. To connect to a sandbox organization that isn't connected, click *Connect*.
++
+The connection process restarts and enables you to connect to the sandbox organization.
```

Comment:
I guess "enables you to" is important?

Reply:
If this is the same for each step, can you put it in the intro or end? Or use a partial?